### PR TITLE
Add Eventual Consistency Propagation Delay to Alias Resource

### PR DESCRIPTION
**Pull request by jtcul**
*Issue #, if available:*

N/A

*Description of changes:*
* Added 60 second eventual consistency propagation delay to Alias creation, deletion, and updates
* Added 60 second propagation delay to Key deletion handler
* Fix code style so that it matches our check style config

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java [hunk 1]
```diff
@@ -25,9 +25,13 @@ public UpdateHandler(final AliasHelper aliasHelper) {
...
-        return proxy.initiate("kms::update-alias", proxyClient, model, callbackContext)
-            .translateToServiceRequest(Translator::updateAliasRequest)
-            .makeServiceCall(aliasHelper::updateAlias)
-            .done(createAliasResponse -> ProgressEvent.defaultSuccessHandler(model));
+        return ProgressEvent.progress(model, callbackContext)
+            .then(
+                progress -> proxy.initiate("kms::update-alias", proxyClient, model, callbackContext)
+                    .translateToServiceRequest(Translator::updateAliasRequest)
+                    .makeServiceCall(aliasHelper::updateAlias)
+                    .done(createAliasResponse -> progress))
```
Comment (nigamaviral) [inline alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java:33]: UpdateAliasResponse ?

Comment (nigamaviral) [reply alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java:33]: Do we want to log here as well ?

File: alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java [hunk 2]
```diff
@@ -38,4 +40,20 @@ public BaseHandlerStd(final AliasHelper aliasHelper) {
       CallbackContext callbackContext,
       ProxyClient<KmsClient> proxyClient,
       Logger logger);
+
+  /**
+   * Perform the final propagation delay to make sure the latest
+   * changes to the alias are available throughout the region.
```
Comment (nigamaviral) [inline alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java:46]: Seems like this function is being used for more than just aliases

File: alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java (hunk 1 above)
Comment (nigamaviral) [inline alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java:33]: UpdateAliasResponse ?

Comment (nigamaviral) [inline alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java:33]: Should we want to log here as well ?

Comment (jtcul) [reply alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java]: Fixed

Comment (jtcul) [reply alias/src/main/java/software/amazon/kms/alias/UpdateHandler.java]: Fixed

File: alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java (hunk 2 above)
Comment (jtcul) [reply alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java:46]: Nope, it's just being used for aliases. There is another version of this function in the Key package, that is used for the Key handlers.

**Review by nigamaviral — APPROVED** [review]

File: alias/.rpdk-config [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
+    "artifact_type": "RESOURCE",
```
Comment (pdz8) [inline alias/.rpdk-config:2]: What's the purpose of the changes in this file?

File: alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java [hunk 4]
```diff
@@ -8,6 +8,8 @@
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 
 public abstract class BaseHandlerStd extends BaseHandler<CallbackContext>  {
+  protected static final int CALLBACK_DELAY_SECONDS = 60;
```
Comment (pdz8) [inline alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java:11]: nit: Should we fix indentation in this file?

**Review by pdz8 — APPROVED** [review]

File: alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java (hunk 4 above)
Comment (jtcul) [reply alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java:11]: Fixed

File: alias/.rpdk-config (hunk 3 above)
Comment (jtcul) [reply alias/.rpdk-config:2]: This file is generated by the CFN java plugin. The update here was the result of an update to the plugin.
